### PR TITLE
init_dbserver postgresql.conf.template load bdr shared library first

### PR DESCRIPTION
**Pull request by mw2q**
Per documentation.

### CODE REVIEW

**Comment by mw2q** [comment]
rebased with fixes to github runner fixes

File: roles/install_dbserver/tasks/EPAS_Debian_install.yml [hunk 1]
```diff
@@ -58,8 +58,8 @@
 - name: "Install PGD packages for EPAS >= 14"
   ansible.builtin.package:
     name:
-      - edb-bdr{{ pgd_version }}-{{ pg_type | lower }}{{ pg_version }}
-      - edb-bdr{{ pgd_version }}-{{ pg_type | lower }}{{ pg_version }}-debuginfo
+      - edb-bdr{{ pgd_version }}-{{ pg_type | lower }}{{ pg_version }}{% if pgd_semver is defined %}={{ pgd_semver }}{% endif %}
+
```
Comment (hannahms) [inline roles/install_dbserver/tasks/EPAS_Debian_install.yml:62]: remove blank line

File: roles/install_dbserver/tasks/PG_Debian_install.yml [hunk 2]
```diff
@@ -37,8 +37,8 @@
 - name: "Install PGD packages for PG >= 14"
   ansible.builtin.package:
     name:
-      - edb-bdr{{ pgd_version }}-{{ pg_type | lower }}{{ pg_version }}
-      - edb-bdr{{ pgd_version }}-{{ pg_type | lower }}{{ pg_version }}-debuginfo
+      - edb-bdr{{ pgd_version }}-{{ pg_type | lower }}{{ pg_version }}{% if pgd_semver is defined %}={{ pgd_semver }}{% endif %}
+
```
Comment (hannahms) [inline roles/install_dbserver/tasks/PG_Debian_install.yml:41]: remove blank line

**Review by hannahms — CHANGES_REQUESTED** [review]
Remove the two empty lines :)

**Comment by mw2q** [comment]
forced pushed corrections

**Pull request merged**